scripts/deploy/contracts/BillBuster.ts

```diff
@@ -0,0 +1,26 @@
+import { deployContract, sleep } from "../utils";
+import { BillBuster } from "../../../build/typechain";
+import { verifyOnEtherscan } from "../../tasks/verifyContractsEtherscan";
+import hre, { ethers } from "hardhat";
+
+export const contractNames = () => ["BillBuster"];
+
+export const constructorArguments = () => [
+  "0xd9BAcC5BccAd9A380001d41Cd234b4D5f33ece76"
+];
+
+export const deploy = async (deployer, setAddresses) => {
+  console.log("deploying BillBuster");
```

Comment:
Want to leave in all these console.logs?

Reply:
Yeah, the deployment process is still quite manual and I find these console.logs help ensure you are running the correct commands